WcaOnRails/app/assets/javascripts/persons.js

```diff
@@ -1,6 +1,10 @@
 // Add params from the search fields to the bootstrap-table for on Ajax request.
 var personsTableAjax = {
   queryParams: function(params) {
+    if (personsTableAjax.queriesOK !== true) {
```

Comment:
Nit: I'd probably have written this as `if (!personsTableAjax.queriesOK)... `, but totally a personal preference thing.